tests/integration/test_poly.py

```diff
@@ -232,7 +232,7 @@ def main() -> None:
 def test_linear(validate):
     module = GuppyModule("test")
     module.load_all(quantum)
-    T = guppy.type_var("T", linear=True, module=module)
+    T = guppy.type_var("T", copyable=False, droppable=False, module=module)
```

Comment:
It would also be good to add tests for `copyable and not droppable` and `not copyable and droppable` type vars